dev/com.ibm.ws.security.wim.core/resources/com/ibm/ws/security/wim/util/resources/WimUtilMessages.nlsprops

```diff
@@ -32,8 +32,8 @@ INVALID_SUPPORTED_ENTITY_TYPE_DEFINITION=CWIMK0001E: The user registry operation
 INVALID_SUPPORTED_ENTITY_TYPE_DEFINITION.explanation= The entity type must be defined with an entity type name and at least one corresponding Relative Distinguished Name (RDN) property.
 INVALID_SUPPORTED_ENTITY_TYPE_DEFINITION.useraction=Modify and correct the entity type definition in the server.xml file.
 
-INVALID_BASE_ENTRY_DEFINITION=CWIMK0002E: The user registry operation could not be completed. The base entry definition {0} is not valid.  Correct the base entry definition in the server.xml file. For example, the syntax is <baseEntry name="..." baseDN="...."/>.
-INVALID_BASE_ENTRY_DEFINITION.explanation= The specified definition of the base entry is incorrect. The syntax for defining a base entry is <baseEntry name="..." baseDN="...."/>.
+INVALID_BASE_ENTRY_DEFINITION=CWIMK0002E: The user registry operation could not be completed. The base entry definition {0} is not valid.  Correct the base entry definition in the server.xml file.
+INVALID_BASE_ENTRY_DEFINITION.explanation= The specified definition of the base entry is incorrect.
```

Comment:
INVALID_BASE_ENTRY_DEFINITION.explanation= The specified base entry definition is invalid.

Reply:
Updated.